components/nexus-repository/src/main/java/org/sonatype/nexus/repository/proxy/CacheInfo.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2008-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.nexus.repository.proxy;
+
+import javax.annotation.Nullable;
+
+import org.joda.time.DateTime;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * Maintains cache details for proxied resources. {@link ProxyFacetSupport} relies on information provided by this
+ * class to implement "aging" and proxy cache invalidation.
+ *
+ * @since 3.0
+ */
+public class CacheInfo
+{
+  private final DateTime lastVerified;
+
+  @Nullable
+  private final String cacheToken;
```

Comment:
After NX3 boot this would be null. For persisting it, see discussion here https://github.com/sonatype/nexus-oss/pull/1502#discussion-diff-33868997

Am for leaving it as is for now, and in "persist cacheToken PR" we can then init it from configuration once persisted (or init it on 1st boot)".